File: MEMO.DAL/Entities/MetaDefinition.cs
using MEMO.DTO.Enums;
using System;
using System.Collections.Generic;
using System.Text;

namespace MEMO.DAL.Entities
{
    public class MetaDefinition : EntityBase
    {
        public string Name { get; set; }
        public MetaType Type { get; set; }

        public Guid DictionaryId { get; set; }
        public Dictionary Dictionary { get; set; }

        public ICollection<TranslationMeta> TranslationMetas { get; set; } = new List<TranslationMeta>();
        public ICollection<MetaDefinitionParameter> MetaDefinitionParameters { get; set; } = new List<MetaDefinitionParameter>();
    }
}

File: MEMO.MobilClient/app/src/main/java/hu/mobilclient/memo/model/MetaDefinition.kt
package hu.mobilclient.memo.model

import com.google.gson.annotations.SerializedName
import hu.mobilclient.memo.model.Enums.MetaType
import java.util.*

class MetaDefinition(@SerializedName("id")var Id: UUID? = null,
                     @SerializedName("name") var Name: String = "",
                     @SerializedName("type") var Type: MetaType = MetaType.TEXT,
                     @SerializedName("metaDefinitionParameters") var MetaDefinitionParameters: MutableList<MetaDefinitionParameter> = ArrayList())